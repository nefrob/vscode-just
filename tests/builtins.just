# Builtin functions

system-info:
  @echo "This is an {{arch()}} machine".

home_dir := env_var('HOME')