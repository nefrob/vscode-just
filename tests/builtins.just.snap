># Builtin functions
#^^^^^^^^^^^^^^^^^^^ source.just comment.line.number-sign.just
>
>system-info:
#^^^^^^^^^^^ source.just entity.name.function.just
#           ^ source.just keyword.operator.recipe.end.just
>  @echo "This is an {{arch()}} machine".
#^^ source.just
#  ^ source.just keyword.operator.quiet.just
#   ^^^^^ source.just
#        ^ source.just string.quoted.double.just
#         ^^^^^^^^^^^ source.just string.quoted.double.just
#                    ^^ source.just string.quoted.double.just string.interpolated.escaping.just string.interpolated.escape.just
#                      ^^^^ source.just string.quoted.double.just string.interpolated.escaping.just support.function.builtin.just
#                          ^^ source.just string.quoted.double.just string.interpolated.escaping.just
#                            ^^ source.just string.quoted.double.just string.interpolated.escaping.just string.interpolated.escape.just
#                              ^^^^^^^^ source.just string.quoted.double.just
#                                      ^ source.just string.quoted.double.just
#                                       ^^ source.just
>
>home_dir := env_var('HOME')
#^^^^^^^^ source.just variable.other.just
#        ^ source.just
#         ^^ source.just keyword.operator.assignment.just
#           ^ source.just
#            ^^^^^^^ source.just support.function.builtin.just
#                   ^ source.just
#                    ^ source.just string.quoted.single.just
#                     ^^^^ source.just string.quoted.single.just
#                         ^ source.just string.quoted.single.just
#                          ^ source.just